javascript-data-model/order-history.js

```diff
@@ -0,0 +1,98 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4th, 2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '114-111111',
+    viewOrderDetails: false,
+    viewinvoice: false,
+    orderDelivered: 'August 8, 2020',
+    image: 'url',
+    returnWindowClosed: true,
+    subTotal: '$31.55',
+    buyAgain: false,
+    viewItem: false,
+    writeReview: false,
+    handedToResident: false,
+    type: [
```

Comment:
This should probably be named something like `items`. Also, you have some data on the item mixed in to your "order" object, such as `image`. Keep the data on the items grouped together, and the data on the order grouped together.

Reply:
This one is still named `type` 👀